src/Domain/CoordinateMaps/Wedge2D.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class Wedge2D.
+
+#pragma once
+
+#include <array>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+
+namespace CoordinateMaps {
+
+/// \ingroup CoordinateMaps
+/// Two dimensional map from the unit square to a wedge, which forms one
+/// quadrant of an annulus. The wedge can be oriented in the +/- x or y axis,
+/// with the inner and outer radii specifiable.
+class Wedge2D {
+ public:
+  static constexpr size_t dim = 2;
```

Comment:
should this be private and/or const as well

Reply:
No

Reply:
The dimension needs to be accessible by the CoordinateMaps implementation and so can't be private, and its constexpr so implicitly const.